qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -166,6 +166,23 @@ def command_accept(self):
         modeman.leave(self._win_id, usertypes.KeyMode.command, 'cmd accept')
         self.got_cmd[str].emit(prefixes[text[0]] + text[1:])
 
+    @cmdutils.register(instance='status-command', scope='window', maxsplit=0)
```

Comment:
Why `maxsplit=0`? This prevents splitting of positional arguments (like with `:bind`) - while we don't have any here, I guess that was unintentional?

Reply:
oops, copy-paste error. Fixed.